ClassPaulita/GuessTheCardGame.cs

```diff
@@ -26,39 +156,55 @@ private bool UserChoseToQuit(char userChoice)
             return userChoice == 'q' || userChoice == 'Q';
         }
 
-        private bool IsUserChoiceValid(char userChoice)
-        {                      
+        private char GetUserGuess(Card previousCard)
+        {
+            Console.Write($"\n\n\tIs the next card greater than {previousCard}(y or n)? ");
+            char userGuess = Convert.ToChar(Console.ReadLine());
 
-            if (UserChoseToPlay(userChoice) || UserchoseToSeeInstructions(userChoice) || UserChosetoSeeBoard(userChoice) || UserChoseToQuit(userChoice))
+            while (!IsUserGuessValid(userGuess))
             {
-                return true;
+                Console.WriteLine("\n\tYour answer is not valid. Please try again!");
+                Console.Write($"\n\n\tIs the next card higher or lower than {previousCard}(y or n)? ");
+                userGuess = Convert.ToChar(Console.ReadLine());
             }
 
-            return false;
+            return userGuess;
         }
 
-        private char GetUserMenuChoice()
+        private void AddUserScoreToBoard(int userScore)
         {
+            Console.WriteLine($"Congratulations! You made it to the Leader Board.\n{_spacing}Please enter your userName (10 characters max):\n\n{_spacing}");
+            string userName = Console.ReadLine();                       
 
-            string menu = $"{spacing}Press P To Play" +
-                          $"{spacing}Press I For Instructions" +
-                          $"{spacing}Press B To See The High Score Board" +
-                          $"{spacing}Press Q To Quit";
+            while (!IsUserNameValid(userName))
+            {
+                Console.WriteLine($"\n{_spacing}You entered an invalid name. Please try again.\n{_spacing}Please enter your userName (10 characters max):\n\n{_spacing}");
```

Comment:
You say there is a max of 10 characters, but I'm not seeing where you are restricting them

Reply:
Actually, the IsUserNameValid method checks if the name is more than 10 characters.